crates/iceberg/src/spec/values.rs

```diff
@@ -31,12 +32,20 @@ use serde_bytes::ByteBuf;
 use serde_json::{Map as JsonMap, Number, Value as JsonValue};
 use uuid::Uuid;
 
-use crate::{Error, ErrorKind};
+use crate::{ensure_data_valid, Error, ErrorKind};
 
 use super::datatypes::{PrimitiveType, Type};
 
+use crate::spec::values::date::{date_from_naive_date, days_to_date, unix_epoch};
+use crate::spec::values::time::microseconds_to_time;
+use crate::spec::values::timestamp::microseconds_to_datetime;
+use crate::spec::values::timestamptz::microseconds_to_datetimetz;
+use crate::spec::MAX_DECIMAL_PRECISION;
 pub use _serde::RawLiteral;
 
+/// Maximum value for [`PrimitiveType::Time`] type.
+const MAX_TIME_VALUE: i64 =
+    23 * 60 * 60 * 1_000_000i64 + 59 * 60 * 1_000_000i64 + 59 * 1_000_000i64 + 999_999i64;
```

Comment:
The const is a bit confusing for the first look. How about `24 * 60 * 60 * 1_000_000i64 - 1`?